object/map.js

```diff
@@ -0,0 +1,11 @@
+
+if (!Object.prototype.map) {
+  Object.defineProperty(Object.prototype, 'map', {
+    value: function(callback) {
+      var object = this
```

Comment:
because I can't use an arrow function the context of `this` changes. I can't just say this[key] in the function anymore